package.json

```diff
@@ -29,10 +27,14 @@
   },
   "license": "Apache-2.0",
   "devDependencies": {
+    "@adobe/spectrum-css-workflow-icons": "^1.0.0-beta.6",
```

Comment:
This dep was moved to the top level before the `yarn install` was added to the bundle release process. It doesn't have to be here, we can move it back (that way, bundles can release with different versions of icons)